### PR TITLE
adding card component

### PR DESCRIPTION
**Pull request by bipin7yadav**
-overlayed text card
-text Card
-dissmiss Card
-Badge Card

### CODE REVIEW

File: card.css [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+    top: 0;
+}
+
+
+.side-nav{
+    text-decoration: none;
+}
+
+.side-nav:hover{
+    background-color: turquoise;
```
Comment (srishti-maurya) [inline card.css:29]: maybe you can use color variables

**Review by srishti-maurya — APPROVED** [review]
You may add a preview link for convenience

**Review by manojkumar8400 — APPROVED** [review]
good to go.

**Comment by bipin7yadav** [comment]
> You may add a preview link for convenience

**Pull request merged**